src/main/java/teammates/common/util/Config.java

```diff
@@ -95,7 +95,11 @@ private String getGcsBucketname() {
     }
 
     private String getAppUrl() {
-        return props.getProperty("app.url");
+        return Url.trimTrailingSlash(props.getProperty("app.url"));
+    }
+    
+    public static AppUrl getAppUrl(String relativeUrl) {
```

Comment:
It will alright, in form of `MalformedURLException` (passed on as `Assumption.fail()`). Yeah I guess I can document about the leading slash requirement.